specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,1430 @@
+          "enum": [
+            "Unspecified",
+            "Kilogram",
+            "Gram",
+            "Milligram",
+            "Gallon",
+            "MetricTon",
+            "Ton",
+            "Pound",
+            "Ounce",
+            "Grain",
+            "PennyWeight",
+            "LongTonBritish",
+            "ShortTonUS",
+            "ShortHundredWeightUS",
+            "Stone",
+            "Dram"
+          ],
+          "x-ms-enum": {
+            "name": "WeightUnit",
+            "modelAsString": true
+          },
+          "description": "The weight Unit of measurement."
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "CurrencyResolution": {
+      "description": "Represents the currency entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "Currency",
+      "properties": {
+        "ISO4217": {
+          "type": "string",
+          "description": "The alphabetic code based on another ISO standard, ISO 3166, which lists the codes for country names. The first two letters of the ISO 4217 three-letter code are the same as the code for the country name, and, where possible, the third letter corresponds to the first letter of the currency name."
+        },
+        "value": {
+          "type": "number",
+          "format": "double",
+          "description": "The money amount captured in the extracted entity"
+        },
+        "unit": {
+          "type": "string",
+          "description": "The unit of the amount captured in the extracted entity"
+        }
+      },
+      "required": [
+        "value",
+        "unit"
+      ]
+    },
+    "BooleanResolution": {
+      "description": "A resolution for boolean expressions",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "Boolean",
+      "properties": {
+        "value": {
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DateTimeResolution": {
+      "description": "A resolution for date/time entity instances.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "DateTime",
+      "properties": {
+        "timex": {
+          "$ref": "#/definitions/TimeExpression"
+        },
+        "dateTimeSubKind": {
+          "type": "string",
+          "enum": [
+            "Time",
+            "Date",
+            "DateTime",
+            "Duration",
+            "Set"
+          ],
+          "x-ms-enum": {
+            "name": "DateTimeSubKind",
+            "modelAsString": false
+          },
+          "description": "The date time SubKind"
```

Comment:
Nit: I see both "date/time" and "date time". Would be ideal if you were consistent for doc purposes.

Reply:
Fixed